src/libs/Localize/LocaleListener/index.desktop.js

```diff
@@ -0,0 +1,7 @@
+import ELECTRON_EVENTS from '../../../../desktop/ELECTRON_EVENTS';
+import BaseLocaleListener from './BaseLocaleListener';
+
+export default {
+    // Send the updated locale to the Electron main process
+    connect: () => BaseLocaleListener.connect(val => window.electron.send(ELECTRON_EVENTS.LOCALE_UPDATED, val)),
```

Comment:
So there is a notable difference between the interface of `libs/Localize/LocaleListener` on desktop v.s: other platforms – web/native LocaleListner's `connect` method works like this:

```js
/**
 * @param {Function} callbackAfterChange
 */
function connect(callbackAfterChange = () => {}) {
    ...
}
```

Whereas on desktop it works like this:

```js
// Note – no way to pass a callback and use the LocaleListener in the renderer process.
// The callback only emits the event to the main process via ipc
function connect() {
    ...
}
```

So we need to patch this up because all libraries need to have the same interface across all platforms:

```js
export default {
    connect: (callbackAfterChange = () => {}) => BaseLocaleListener.connect((val) => {
        // Send the updated locale to the Electron main process
        window.electron.send(ELECTRON_EVENTS.LOCALE_UPDATED, val);
        
        // Then execute the callback provided for the renderer process
        callbackAfterChange(val);
    }),
};
```